crates/cheatnet/src/state.rs

```diff
@@ -36,6 +37,24 @@ pub enum CheatTarget {
     Multiple(Vec<ContractAddress>),
 }
 
+// Specifies the duration of the cheat
+#[derive(Clone)]
+pub enum CheatSpan {
+    Indefinite,
+    Number(usize),
+}
+
+impl Add<usize> for CheatSpan {
```

Comment:
i don't think i see a use-case for this

Reply:
Good catch, I've removed all of them and forgot about it 👍